aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,33 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## November 8 - November 19, 2021
+
+### Communications
+- **Office Hours** - A second, timezone-friendly option for office hours was held with the India team to ensure all questions were answered.
+- **Email** - An e-mail was sent to the India team that directly answered previously asked questions.
+- **Figma Guide** - A fundamental and advanced functionality of Figma to help you understand and leverage the HPE Design System 
+in your [design process.](https://www.figma.com/proto/ehx1youYYwuaGasfwTbkNK/Figma-Guide?page-id=425%3A284&node-id=846%3A1435&viewport=241%2C48%2C0.03&scaling=min-zoom&starting-point-node-id=644%3A590)
+
+### "Big Things" progress updates
+Here are the latest developments regarding roadmap items:
+- **NameValueList**. Now available in [Figma](https://www.figma.com/file/Ny7rYK5gI5VO4PtZsq6vGC/HPE-Name-Value-List?node-id=2%3A93) and [Design System site.](https://design-system.hpe.design/components/namevaluelist?q=name)
+- **Navigation**. Guidance and examples for [Drill Down Navigation on Card, List and Tables.](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1647%3A9622
+)
+- **Button Guidance**.
+Guidance on left or right aligned buttons on forms, modals, layers and wizards [updated in Figma.](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=4071%3A8993)
+
+### Stay current with the latest changes
+- **Icons** - [Removed stroke version of icons from Figma](https://www.figma.com/file/G99KGffGLFKICy8xRs51lW/HPE-Icons?node-id=261%3A0) component library to limit to outline versions only so icons are correctly sized.
+- **Tables** - Guidance and examples on how to show [states and statuses within a table.](https://www.figma.com/file/XEoIWIlLfcARRIFhkpFCch/HPE-DataTable?node-id=7210%3A147386)
+- **Checkbox** - [Align checkbox label when multi-line.](https://github.com/grommet/grommet-theme-hpe/pull/222)
+- **FileInput** - Added an example with a value that needs validation on [Design System site](https://design-system.hpe.design/components/fileinput?q=file#fileinput-validation) and [Figma](https://www.figma.com/file/8HwAiiSDloOTjUxqvCTypq/HPE-FileInput-Component?node-id=1164%3A43348).
+
+### Latest Releases
+- **Grommet** - [Released Grommet 2.19.0](https://github.com/grommet/grommet/releases/tag/v2.19.0)
+- **Grommet-Theme-HPE** - [Released Grommet-theme-hpe v2.3.0](https://github.com/grommet/grommet-theme-hpe/releases/tag/v2.3.0)
```

Comment:
```suggestion
- **Grommet-Theme-HPE** - Released [Grommet-theme-hpe v2.3.0](https://github.com/grommet/grommet-theme-hpe/releases/tag/v2.3.0).
```

In the spirit of "why should I care?" This should be accompanied by any noteworthy features included in the release.